server/aesdsocket.c

```diff
@@ -58,23 +132,222 @@ static void signal_handler (int signo)
     exit (EXIT_SUCCESS);
 }
 
-int main(int argc, char *argv[])
+
+void receive_send_data(void* thread_param)
+{
+
+	thread_data_t *thread_func_args = (thread_data_t*)thread_param;
+
+	int received_bytes, write_bytes, read_bytes, send_bytes;
+	int buf_loc = 0;
+	int realloc_count = 1;
```

Comment:
@Sharath-Jonnala check your editor spacing settings to make your code more readable, it looks like the indent settings change here.